static/usr/libexec/core/get-arg

```diff
@@ -31,7 +31,8 @@ fi
 set --
 
 # We cannot use ANSI-C quoting (e.g. $'\n') in busybox-initramfs
-whitespaces="$(printf '\t\n\v\f\r \xA0')"
+# dash's printf builtin, so we make sure does not handle \x
+whitespaces="$(/usr/bin/printf '\t\n\v\f\r \xA0')"
```

Comment:
I think that hex escape is not part of the standard, you need to use octal.

`printf '\t\n\v\f\r \240`  seems to work in dash.

Reply:
I have tested, `\240` works for all. I have changed the fix.